lib/ui/views/intro/intro_welcome.dart

```diff
@@ -258,3 +81,328 @@ class _IntroWelcomeState extends ConsumerState<IntroWelcome> {
+    return Expanded(
+      child: SingleChildScrollView(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: const <Widget>[
+            _LogoArchethic(),
+            _WelcomTextFirst(),
+            _WelcomTextSecond(),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _LogoArchethic extends ConsumerWidget {
+  const _LogoArchethic();
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final theme = ref.watch(ThemeProviders.selectedTheme);
+    return Center(
+      child: Container(
+        padding: const EdgeInsets.only(
+          left: 20,
+          right: 20,
+        ),
+        child: SizedBox(
+          height: 200,
+          child: AspectRatio(
+            aspectRatio: 3 / 1,
+            child: Image.asset(
+              '${theme.assetsFolder!}${theme.logo!}.png',
+              height: 200,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _WelcomTextFirst extends ConsumerWidget {
+  const _WelcomTextFirst();
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final theme = ref.watch(ThemeProviders.selectedTheme);
+    final localizations = AppLocalization.of(context)!;
+    return Container(
+      margin: const EdgeInsets.only(
+        right: 20,
+        left: 20,
+      ),
+      child: AutoSizeText(
+        localizations.welcomeText,
+        maxLines: 3,
+        style: theme.textStyleSize16W400Primary,
+      ),
+    );
+  }
+}
+
+class _WelcomTextSecond extends ConsumerWidget {
+  const _WelcomTextSecond();
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final theme = ref.watch(ThemeProviders.selectedTheme);
+    final localizations = AppLocalization.of(context)!;
+    return Container(
+      margin: const EdgeInsets.only(
+        top: 20,
+        right: 20,
+        left: 20,
+      ),
+      child: AutoSizeText(
+        localizations.welcomeText2,
+        maxLines: 5,
+        style: theme.textStyleSize12W100Primary,
+      ),
+    );
+  }
+}
+
+class _Footer extends ConsumerWidget {
+  const _Footer({
+    required this.isConnectivityAvailable,
+    required this.checkedValue,
+    required this.onToggleCGU,
+  });
+
+  final bool isConnectivityAvailable;
+  final bool checkedValue;
+  final Function(bool? newValue) onToggleCGU;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Column(
+      children: <Widget>[
+        _CGU(
+          isConnectivityAvailable: isConnectivityAvailable,
+          checkedValue: checkedValue,
+          onToggleCGU: onToggleCGU,
+        ),
+        const SizedBox(
+          height: 10,
+        ),
+        _ButtonNewWallet(
+          checkedValue: checkedValue,
+        ),
+        _ButtonImportWallet(
+          checkedValue: checkedValue,
+        ),
+      ],
+    );
+  }
+}
+
+class _CGU extends ConsumerWidget {
```

Comment:
Continue your idea of structuration: separate CGU and version info.
So don't use isConnectivityAvailable in param but directly a condition to display or not the widget ``_CGU``